config/deploy.rb

```diff
@@ -1,24 +1,65 @@
-namespace :paperclip do
-  desc 'Create a storage folder for Paperclip attachment in shared path'
-  task :create_storage do
-    run 'mkdir -p #{shared_path}/storage'
-  end
+# config valid only for Capistrano 3.1
+lock '3.1.0'
 
-  desc 'Link the Paperclip storage folder into the current release'
-  task :link_storage do
-    run 'ln -nfs #{shared_path}/storage #{release_path}/storage'
-  end
-end
+set :application, 'fsek'
+set :repo_url, 'git@github.com:fsek/web.git'
+
+# what specs should be run before deployment is allowed to
+# continue, see lib/capistrano/tasks/run_tests.cap
+set :tests, []
+
+# Default branch is :master
+# ask :branch, proc { `git rev-parse --abbrev-ref HEAD`.chomp }
+
+# Default deploy_to directory is /var/www/my_app
+# set :deploy_to, '/var/www/my_app'
+
+# Default value for :scm is :git
+# set :scm, :git
+
+# Default value for :format is :pretty
+# set :format, :pretty
+
+# Default value for :log_level is :debug
+# set :log_level, :debug
 
-before 'deploy:setup', 'paperclip:create_storage'
-after 'deploy:update_code', 'paperclip:link_storage'
+# Default value for :pty is false
+# set :pty, true
 
-# Symlink the database
-after 'deploy:update_code', 'deploy:symlink_db'
+# Default value for :linked_files is []
+# set :linked_files, %w{config/database.yml}
+
+# Default value for linked_dirs is []
+# set :linked_dirs, %w{bin log tmp/pids tmp/cache tmp/sockets vendor/bundle public/system}
+
+# Default value for default_env is {}
+# set :default_env, { path: "/opt/ruby/bin:$PATH" }
+
+# Default value for keep_releases is 5
+# set :keep_releases, 5
 
 namespace :deploy do
-  desc 'Symlinks the database.yml'
-  task :symlink_db, roles: 'app' do
-    run 'ln -nfs #{deploy_to}/shared/config/database.yml #{release_path}/config/database.yml'
+  before :deploy, "deploy:check_revision"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.